### DISCUSSION TITLE
Currently viewed page auto-scroll in sidebar

### DISCUSSION THREAD

**Discussion by matthew-ant** [outline/outline#7931] [Ideas]
We often have related pages near each other, using the sidebar for navigation.
Our sidebar is long enough that finding the current page is difficult! Would be great to have a button to do this for you.

**Comment by tommoor** [comment]
I'm not sure what you mean by localize, the currently opened document should already be scrolled into view automatically

**Comment by matthew-ant** [comment]
For me, the sidebar does not scroll automatically at all

**Reply by tommoor** [reply]
Could be a bug here – if you can post a video of the behavior that might help track it down
